lib/database/repositories/QcFlagRepository.js

```diff
@@ -26,6 +36,93 @@ class QcFlagRepository extends Repository {
         super(QcFlag);
     }
 
+    /**
+     * Find IDs of QC flags in GAQ effective periods for given data pass and runs
+     *
+     * @param {number} scope.dataPassId id of data pass id
+     * @param {number[]} [scope.runNumbers] list of run numbers
+     * @return {Promise<GaqEffectivePeriods>} promise resolvesd once periods are calculated
+     */
+    async findFlagsInGaqEffectivePeriods({ dataPassId, runNumbers }) {
```

Comment:
If this function is named `findFlagsInGqaEffectivePeriods` it should return QcFlags or SequelizeQcFlags?

Reply:
renamed